renovate.json

```diff
@@ -1,6 +1,25 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:recommended"
-  ]
+    "config:base",
+    "schedule:weekly",
+    ":semanticCommits"
+  ],
+  "postUpdateOptions": [
+    "gomodTidy"
+  ],
+  "commitMessagePrefix": "chore(all): ",
+  "commitMessageAction": "update",
+  "groupName": "all",
+  "packageRules": [
+    {
+      "updateTypes": [
+        "major"
+      ],
+      "enabled": false
```

Comment:
If I'm reading this correctly, this is grouping all updates and disabling major updates. It may be grouping all updates except major updates which are left individual, which seems like a better path. Either way seems like a good iteration.